eng/pipelines/common/evaluate-changed-paths.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  subsetName: ''
+  arguments: ''
+
+steps:
+  - ${{ if ne(parameters.arguments[0], '') }}:
+    - script: eng/pipelines/evaluate-changed-paths.sh
+              -azurevariable containschange
```

Comment:
The script appears to always set the variable with the name `containsChange` but the consuming script assume it will be `SetPathVars_{subsetName}.containsChange`. What am I missing here? Assuming this is just part of my ignorance in pipelines.

Reply:
Yeah this is because of the way that azure devops job variables work. So basically if you have a job variable, it will be an output variable. So you can have multiple variables with the same name coming from different jobs because the way you access it is via dependencies.JobName.outputs[`StepName.VarName`];

Since the StepName has to have a unique name, I named it as `SetPathVars_{subsetName}`, so the variable name can be the same for all, that makes it easier and more consistent.

https://github.com/dotnet/runtime/pull/748/files#diff-82029ac3022fe163333fbd15b295bef6R11

Reply:
I see. The variable name is not enough, has to be step name qualified.

Reply:
Correct, since it is an output variable.